doc/whats_new/v0.23.rst

```diff
@@ -159,6 +159,17 @@ Changelog
   instead of predictions.
   :pr:`14848` by :user:`Venkatachalam N <venkyyuvy>`.
 
+- |API| Deprecated public attributes `standard_coef_`, `standard_intercept_`,
+  `average_coef_`, and `average_intercept_` in
+  :class:`linear_model.BaseSGD`,
+  :class:`linear_model.BaseSGDClassifier`,
+  :class:`linear_model.SGDClassifier`,
+  :class:`linear_model.BaseSGDRegressor`,
```

Comment:
All the base models (i.e. BaseSGD, BaseSGDClassifier, BaseSGDRegressor),  are not in the public API (i.e. not imported in `linear_model.__init__`), and so shouldn't appear here.

Reply:
OK! Yes...make sense!

Reply:
Done! Thanks!